src/System.Management.Automation/security/nativeMethods.cs

```diff
@@ -1152,8 +1154,15 @@ DWORD idxCert
         ///psiginfo: SIGNATURE_INFO*
         ///ppCertContext: void**
         ///phWVTStateData: HANDLE*
-        [DllImportAttribute("wintrust.dll", EntryPoint = "WTGetSignatureInfo", CallingConvention = CallingConvention.StdCall)]
-        internal static extern int WTGetSignatureInfo([InAttribute()][MarshalAsAttribute(UnmanagedType.LPWStr)] string pszFile, [InAttribute()] System.IntPtr hFile, SIGNATURE_INFO_FLAGS sigInfoFlags, ref SIGNATURE_INFO psiginfo, ref System.IntPtr ppCertContext, ref System.IntPtr phWVTStateData);
+        // [DllImportAttribute("wintrust.dll", EntryPoint = "WTGetSignatureInfo", CallingConvention = CallingConvention.StdCall)]
```

Comment:
Please move out of native since it is now managed.  Also if only used once, can just inline.